src/views/partials/__meta_header.njk

```diff
@@ -1,7 +1,14 @@
 <head>
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta charset="utf-8"/>
-    <title>{{ title if title else lang.page_header + lang.title_end }}</title>
+    <title>
+        {% if loggedInUserRole === UserRole.OWNER or loggedInUserRole === UserRole.ADMIN %}
+            {{ lang.page_header }}
+        {% else %}
+            {{ lang.page_header_standard }}
+        {% endif %}
+        {{ lang.title_end }}
+    </title>
```

Comment:
I don't think we should modify this code, as `__meta_header.njk` is shared amongst multiple files through inheritance:

https://github.com/companieshouse/acsp-manage-users-web/blob/2b8bc29e7ce2c0059170826281e25c38a165bb44/src/views/layouts/default.njk#L7

<img width="643" alt="Screenshot 2024-08-02 at 10 11 50" src="https://github.com/user-attachments/assets/5f80714b-e624-4dfc-9cef-d9c208313e7f">

So 6 of the other pages might break due to not having `page_header_standard` on their translation files. 

An alternative fix would be reverting this code back to normal, and creating a `title` on the back-end we can pass to the template directly, so it would just work. 

i.e

```diff
diff --git a/src/routers/controllers/manageUsersController.ts b/src/routers/controllers/manageUsersController.ts
--- a/src/routers/controllers/manageUsersController.ts	(revision 1189fba02e0a0b9abeefc8fc931c43c71390827d)
+++ b/src/routers/controllers/manageUsersController.ts	(date 1722590068840)
@@ -59,7 +59,9 @@
         }
     });
 
+    const title = "" // form title here.  based on translations/role logic.
     return {
+        title: title,
         lang: translations,
         backLinkUrl: constants.DASHBOARD_FULL_URL,
         addUserUrl: constants.ADD_USER_FULL_URL + constants.CLEAR_FORM_TRUE,
```